docs/4.2.yaml

```diff
@@ -43,34 +43,34 @@ nav:
         - Admin Manual: admin-guide.md
         - Production Guide: production.md
         - FAQ: faq.md
-    - Teleport Advanced Features: 
-        - Enhanced Session Recording: features/enhanced_session_recording.md
-        - Using Teleport with PAM: features/ssh_pam.md
     - Infrastructure Guides:
         - AWS: aws_oss_guide.md
         - AWS HA with Terraform: aws_terraform_guide.md
         - GCP: gcp_guide.md
         - IBM: ibm_cloud_guide.md
         - Kubernetes Guide: kubernetes_ssh.md
-    - Enterprise Guides:
+    - Teleport Enterprise :
         - Introduction: enterprise/index.md
         - Quick Start Guide: enterprise/quickstart-enterprise.md
         - Single sign-on (SSO): enterprise/ssh_sso.md   
         - FedRAMP & FIPS: enterprise/ssh_fips.md        
         - RBAC: enterprise/ssh_rbac.md 
-    - SSO Guides:
-        - Azure Active Directory (AD): ssh_azuread.md
-        - Active Directory (ADFS): ssh_adfs.md
-        - G Suite: ssh_gsuite.md
-        - OneLogin: ssh_one_login.md
-        - OIDC: oidc.md
-        - Okta: ssh_okta.md
+        #- SSO Guides:
+        #    - Azure Active Directory (AD): enterprise/sso/ssh_azuread.md
+        #    - Active Directory (ADFS):  enterprise/sso/ssh_adfs.md
+        #    - G Suite:  enterprise/sso/ssh_gsuite.md
+        #    - OneLogin:  enterprise/sso/ssh_one_login.md
+        #    - OIDC:  enterprise/sso/oidc.md
+        #    - Okta:  enterprise/sso/ssh_okta.md
```

Comment:
I think this is too hidden now, it becomes too difficult to find the provider-specific guides. We should probably move these under the "Teleport Enterprise" header to make them easier to locate at a quick glance, unless you have a bigger idea for the whole thing?

Reply:
We do a bigger project in the works and for now we want to focus on the community edition and the current sidebar is project too much Enterprise product. We'll address this soon with an upcoming release.